src/main/kotlin/biokotlin/featureTree/Graph.kt

```diff
@@ -846,11 +847,11 @@ internal class Graph private constructor(
 
             val graph = Graph(multipleParentage)
             modifySchema?.invoke(graph.schema)
-            FileReader(file).useLines { lines ->
+            bufferedReader(file).useLines { lines ->
                 var lineCounter = 0
                 for (line in lines) {
                     lineCounter++
-
+                    if (line.isEmpty() || line.isBlank()) continue //skip blank lines
```

Comment:
I guess useLines() would never return a null?

Reply:
correct.  This compiles fine so its not nullable.